src/Panel.js

```diff
@@ -166,7 +174,7 @@ const Panel = React.createClass({
       if (this.props.collapsible) {
         header = cloneElement(header, {
           className,
-          children: this.renderAnchor(header.props.children)
+          children: this.renderAnchor(header.props.children, headerRole)
```

Comment:
The use-case we're anticipating here is using an accordion like a set of tabs, as in the example in the react-bootstrap documentation. Based on our research, the `a` is the correct place to put the role attribute. Here's [one blog post that confirms this](https://www.marcozehe.de/2013/02/02/advanced-aria-tip-1-tabs-in-web-apps/).